src/main/java/com/suse/saltstack/netapi/event/EventStream.java

```diff
@@ -129,99 +126,107 @@ public void removeEventListener(EventListener listener) {
     }
 
     /**
-     * Helper method that returns the current number of subscribed
-     * listeners.
+     * Helper method that returns the current number of subscribed listeners.
+     *
      * @return The current number listeners.
      */
     public int getListenerCount() {
-        return listeners.size();
+        int size = 0;
```

Comment:
Hi. I'm realizing that you solution is safer not for styling code but for the variable out of synchronized block. I'm fixing it. Thank you.